src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/lora_pattern.cpp

```diff
@@ -126,9 +126,10 @@ static std::string getTestCaseName(const testing::TestParamInfo<LoraPatternParam
         ASSERT_TRUE(inferRequestRef);
 
         generate_inputs(targetStaticShapes.front());
-        for (const auto& input : inputs) {
-            inferRequest.set_tensor(input.first, input.second);
-            inferRequestRef.set_tensor(input.first, input.second);
+        for (const auto& [port, tensor] : inputs) {
+            // Use read-only tensors as inputs, created from `const void*`
+            inferRequest.set_tensor(port, {tensor.get_element_type(), tensor.get_shape(), tensor.data()});
```

Comment:
```suggestion
            inferRequest.set_tensor(port, {tensor.get_element_type(), tensor.get_shape(), std::as_const(tensor).data()});
```

Reply:
Is not required as `tensor` is const and return const pointer.